dll/win32/shell32/lang/ja-JP.rc

```diff
@@ -64,9 +64,9 @@ BEGIN
     POPUP ""
     BEGIN
         MENUITEM SEPARATOR
-        MENUITEM "切り取り(&U)", IDM_CUT
+        MENUITEM "切り取り(&T)", IDM_CUT
         MENUITEM "コピー(&C)", IDM_COPY
-        MENUITEM "貼り付け(&P)", IDM_INSERT
+        MENUITEM "貼り付け", IDM_INSERT
```

Comment:
Why deleted `(&P)`?

Reply:
read the commit comment. It would collide with MENUITEM "プロパティ(&P)", IDM_PROPERTIES
I did the same as was done earlier for en-US, where the accel was stripped for the IDM_INSERT.

Reply:
![image](https://github.com/reactos/reactos/assets/33393466/02b063b7-3dfb-40c2-b8a9-4bde671ef372)
![image](https://github.com/reactos/reactos/assets/33393466/a22c9325-6f99-4299-8c79-290ba6c1d421)

Reply:
I left a comment how to structurally solve that mess in the future within en-US.rc

Reply:
![image](https://github.com/reactos/reactos/assets/33393466/91e5f250-ea42-49e4-9453-be4a1bc5b7dd)